sopel/config/types.py

```diff
@@ -423,7 +427,7 @@ class ListAttribute(BaseValidated):
     :param str name: the attribute name to use in the config file
     :param strip: whether to strip whitespace from around each value
                   (optional; applies only to legacy comma-separated lists;
-                  multi-line lists are always stripped)
+                  multi-line lists are always stripped; deprecated)
```

Comment:
I would suggest putting "deprecated" earlier in the string, close to "optional", as both are very important statuses.

Maybe "(optional, deprecated; applies […]"?

----

Also, since I can't line-note `__init__` because it's outside the diff… Have you thought ahead to what it will be like in 9.0 when we actually remove the behavior? Got any more sophisticated plan for removing `strip` than just letting the constructor `TypeError` if a plugin tries to use this unexpected kwarg after removal?

Reply:
I think we can raise a warning if `strip is not None` in Sopel 9, and remove it in Sopel 10.